vignettes/examples/cifar10_densenet.R

```diff
@@ -0,0 +1,82 @@
+#' In this example we will train a DenseNet-40-12 to classify images from the 
```

Comment:
Maybe link to the DenseNet paper and provide a sentence regarding motivation for DenseNet?